src/Microsoft.ML.HalLearners/OlsLinearRegression.cs

```diff
@@ -87,7 +87,10 @@ internal OlsLinearRegressionTrainer(IHostEnvironment env, Options options)
         protected override RegressionPredictionTransformer<OlsLinearRegressionModelParameters> MakeTransformer(OlsLinearRegressionModelParameters model, Schema trainSchema)
              => new RegressionPredictionTransformer<OlsLinearRegressionModelParameters>(Host, model, trainSchema, FeatureColumn.Name);
 
-        public RegressionPredictionTransformer<OlsLinearRegressionModelParameters> Train(IDataView trainData, IPredictor initialPredictor = null)
+        /// <summary>
+        /// Continues the training of a <see cref="OlsLinearRegressionTrainer"/> using an initial predictor and returns a <see cref="ITransformer"/>.
+        /// </summary>
+        public RegressionPredictionTransformer<OlsLinearRegressionModelParameters> Fit(IDataView trainData, IPredictor initialPredictor)
```

Comment:
>IPredictor [](start = 108, length = 10)

Can we file issue, to clean `IPredictor` from public interface? @TomFinley trying to hide them in [https://github.com/dotnet/machinelearning/issues/2251](https://github.com/dotnet/machinelearning/issues/2251) and we should accept strictly typed classes, like `LinearModelParameters` here.

Reply:
[https://github.com/dotnet/machinelearning/issues/2553](https://github.com/dotnet/machinelearning/issues/2553)

---
In reply to: [256625522](https://github.com/dotnet/machinelearning/pull/2528#discussion_r256625522) [](ancestors = 256625522)